bower.json

```diff
@@ -28,13 +28,27 @@
     "maps"
   ],
   "dependencies": {
-    "polymer": "Polymer/polymer#^1.7.1",
-    "google-apis": "GoogleWebComponents/google-apis#^1.1.7",
-    "iron-resizable-behavior": "PolymerElements/iron-resizable-behavior#^1.0.5",
-    "iron-selector": "PolymerElements/iron-selector#^1.5.2"
+    "polymer": "Polymer/polymer#1.9 - 2",
+    "google-apis": "GoogleWebComponents/google-apis#2.0-preview",
```

Comment:
@e111077 Oh, this should be `google-apis#1 - 2` instead of `google-apis#^2.0.0`. Could you please change it and bump version of `google-map` to `2.0.1`?